CODE_OF_CONDUCT.md

```
@@ -0,0 +1,23 @@
+<div align="center" width="100%"><img src="./assets/images/open-communities.png"></div>
+
+[_SparkForge Development_](https://sparkforge.dev) takes pride in creating quality open code and open communities. We are committed to providing a welcoming and inspiring community where our members create amazing projects and help each other grow.
+
+## Our open communities strive to:
```

Markdownlint

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)